adlfs/transfer.py

```diff
@@ -217,11 +215,9 @@ class ADLTransferClient(object):
     adlfs.multithread.ADLUploader
     """
 
-    DEFAULT_TMP_PATH = 'tmp'
-
     def __init__(self, adlfs, name, transfer, merge=None, nthreads=None,
-                 chunksize=2**28, blocksize=2**25, tmp_path=DEFAULT_TMP_PATH,
-                 tmp_unique=True, persist_path=None, delimiter=None):
+                 chunksize=2**28, blocksize=2**25, chunked=True,
```

Comment:
Do we expect the defaults ever not to be used? Or is this just included here for testing?